index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/home/safeez/Desktop/Portfolio/Portfolio-Website/style.css">
+    <title>Portfolio-Website</title>
+</head>
+<body>
+    <div class="header container">
+        <div class="header one">My Logo</div>
+
+        <div class="header menu-icon">
+            <img src="src/Icon-Menu.png" alt="menu"/>
+        </div>
+    </div>
+
+    <section class="heading">
+        <h1 class="title">I’m Shaheda
+            Glad to see you!</h1>
+    </section>
+
+    <section class="paragraph">
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    </section>
+
+    <p class="line">LET’S CONNECT</p>
+
+    <section class="social-media-icons">
+        <img class="twitter" src="src/Twitter.png" alt="twitter-icon"/>
+
+        <img class="linkedin-icon" src="src/Linkedin-icon.png" alt="linkedin-icon"/>
+
+        <img class="msign" src="src/M.png" alt="Micon"/>
+
+        <img class="github" src="src/Github.png" alt="github-icon"/>
+
+        <img class="finger" src="src/Finger.png" alt="finger-icon"/>
+    </section>
+    
+</body>
```

Comment:
- [ ] Please make sure you have a proper background image for the headline section as specified in the design.
- [ ] Kindly make sure that your design matches the original design's margins, fonts, and colors.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/153552460-17baeeb2-8233-41cb-8b1a-4b1a2cf9fc4f.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/153552624-91060d60-11bb-4e60-8141-816c9dc2b6f2.png)